doc-Appliance_Hardening_Guide/topics/Appliance_Security.adoc

```diff
@@ -0,0 +1,11 @@
+= Appliance Security
+
+:leveloffset: 2
```

Comment:
It is not clear to me why you want to offset headers by 2 here. Normally this is only useful when you are nesting sub-topics and their sub-topics together into a skeleton document.